Service/IWorkoutService.cs

```diff
@@ -8,10 +8,30 @@ namespace FitnessTrackerApi.Service;
 /// </summary>
 public interface IWorkoutService
 {
+    
+    /// <summary>
+    /// Получает все тренировки.
+    /// </summary>
```

Comment:
```suggestion
    /// </summary>
    /// <param ...>
```
Точно также добавить параметры как и выше писал